src/test/java/teammates/test/cases/action/InstructorFeedbackResultsDownloadActionTest.java

```diff
@@ -163,6 +167,57 @@ public void testExecuteAndPostProcess() throws Exception {
         expectedFileName = session.getCourseId() + "_" + session.getFeedbackSessionName();
         assertEquals(expectedFileName, result.getFileName());
         verifyFileContentForDownloadWithMissingResponsesHidden(result.getFileContent(), session);
+
+        ______TS("Typical case: results downloadable by question");
```

Comment:
Should you add 'not within section' to be symmetric with the counterpart `TS` below?